cmd/node/config/external.toml

```diff
@@ -44,13 +44,16 @@
 [HostDriverConfig]
     # This flag shall only be used for observer nodes
     Enabled = false
-    # This flag will start the WebSocket connector as server or client
-    IsServer = false
-    # The url of the WebSocket client/server
+    # This flag will start the WebSocket connector as server or client( can be "client" or "server")
```

Comment:
typo:
```
as server or client (can be "client"
```
wrong spacing near the `(` character

Reply:
done